Source/WarpX.cpp

```diff
@@ -1163,6 +1156,15 @@ WarpX::ReadParameters ()
         // Overwrite update_with_rho with value set in input file
         pp_psatd.query("update_with_rho", update_with_rho);
 
+        if (do_dive_cleaning == true && update_with_rho == false)
+        {
+            update_with_rho = true;
+            RecordWarning(
+                "PSATD",
+                "Setting psatd.update_with_rho = 1 because warpx.do_dive_cleaning = 1",
+                WarnPriority::low);
+        }
```

Comment:
Note: Based on recent discussions, I'll replace this warning and automatic replacement with an abort.